docs/book/component-guide/step-operators/azureml.md

```diff
@@ -15,10 +15,17 @@ You should use the AzureML step operator if:
 
 ### How to deploy it
 
-* Create a `Machine learning` [resource on Azure](https://docs.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources) .
-* Once your resource is created, you can head over to the `Azure Machine Learning Studio` and [create a compute cluster](https://docs.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources#cluster) to run your pipelines.
-* Create an `environment` for your pipelines. Follow [this guide](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-in-studio) to set one up.
-* (Optional) Create a [Service Principal](https://docs.microsoft.com/en-us/azure/developer/java/sdk/identity-service-principal-auth) for authentication. This is required if you intend to run your pipelines with a remote orchestrator.
+{% hint style="info" %}
+Would you like to skip ahead and deploy a full ZenML cloud stack already,
+including an AzureML step operator? Check out the [in-browser stack deployment wizard](../../how-to/stack-deployment/deploy-a-cloud-stack.md),
```

Comment:
Is this possible? AFAIK, step operators are not supported by the 1-click deployment or the stack wizard, right?

Reply:
they will be starting with the next release :) I added step operators to all of our terraform modules